components/core/header/nav-bar/NavBarItemDropdown.vue

```diff
@@ -95,4 +100,31 @@ export default {
     @apply inline-flex w-full h-full justify-center items-center bg-transparent;
     z-index: 100;
 }
+.core-menu-fade-enter-active {
+    transition: all 0.2s;
+}
+.core-menu-fade-leave-active {
+    transition: all 0.5s;
+    transition-delay: 0.2s;
+}
+.core-menu-fade-enter,
+.core-menu-fade-leave-to {
+    opacity: 0;
+}
+.menu-mask {
+    @apply absolute w-full h-0 top-full;
+    background: #121023;
+}
+.core-menu-mask-enter-active {
+    transition: all 0.2s;
+}
+.core-menu-mask-leave-active {
+    transition: all 0.5s;
+    transition-delay: 0.2s;
+}
+.core-menu-mask-enter,
+.core-menu-mask-leave-to {
+    height: 100%;
+    top: 0px;
+}
```

Comment:
These are nit picking but will look cleaner:
1. Please try to use TailwindCSS as much as possible: `opacity-0`, `top-0`, `h-full`, etc. Note that `0px` is actually redundant as zero doesn't need a unit.
2. Replace `background` with `background-color` (I saw the project mix both but `background-color` is more precise.)

Reply:
1. That's right, thanks.
2. Which is better? Should I use `background-color: #121023;` in CSS, or use TailwindCSS like `bg-[#121023]` as a class ?(Though the second way haven't been used in the whole project.)

Reply:
2. Let's use the former for now. Eventually (when?! Lol), we should change the tailwind config's theme settings. If you'd like to introduce the arbitrary usage, you're welcome to create another pr for refactoring it 😆. Or updating the theme will be even better!

Reply:
Got it, ha!